tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2/1.md

```diff
@@ -0,0 +1,25 @@
+# Module sayakpaul/deeplabv3-mobilenetv2/1
+Lightweight deep learning model for semantic image segmentation.
+
+<!-- asset-path: https://github.com/sayakpaul/Adventures-in-TensorFlow-Lite/releases/download/v0.1.0/mobilenetv2_coco_voctrainval_tflite.tar.gz -->
+<!-- module-type: image-segmentation -->
+<!-- network-architecture: DeepLab (mobilenetv2_coco_voc_trainval) -->
+<!-- dataset: PASCAL VOC 2012 -->
+<!-- fine-tunable: false -->
+<!-- language: en -->
+<!-- format: tflite -->
```

Comment:
The `format` is not required for Lite models.

Reply:
@maringeo this error trace seems weird then:

https://github.com/tensorflow/hub/pull/610/checks?check_run_id=784513028

Reply:
Thanks for spotting that. It's because the validation test was not compatible to run on Lite MD files, so this assertion was incorrectly applied. I removed the assertion for Lite models so all tests pass now.